src/configs/buildConfig.js

```diff
@@ -38,7 +38,7 @@ export const buildEnvironment = {
   STORYLY_TOKEN: '_storyly_token_',
   KOCHAVA_IOS_ID: '_kochava_ios_id_',
   KOCHAVA_ANDROID_ID: '_kochava_android_id_',
-  PROJECT_ID: '15fcfb7323fcce5aa1b58afe4dc6d847',
+  WALLETCONNECT_PROJECT_ID: '15fcfb7323fcce5aa1b58afe4dc6d847',
```

Comment:
Sorry just remember to change `15fcfb7323fcce5aa1b58afe4dc6d847` to `_wallet_connect_id_` here

Reply:
Updated it!